docs/notebooks/markov-models.ipynb

```diff
@@ -137,7 +137,7 @@
     "The symbol etymology, which usually gets swept under the rug in mathematically-oriented papers, are as follows: \n",
```

Comment:
<p>Here also, it would be really nice to see you take care of such modelling issues</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ericmjl/bayesian-analysis-recipes/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='102'/>

Reply:
<p>I suspect if I fix the chain initialization problem, then everything else should be fine and the two sentence paragraph wouldn't be necessary.</p>